xml/storage_multipath.xml

```diff
@@ -4643,6 +4643,32 @@ Give root password for maintenance
      </para>
     </listitem>
    </itemizedlist>
+   <important>
+    <para>
+     At this stage, you are working in a <command>dracut</command> emergency
+     shell inside initrd. All modification resulting from the following
+     procedures will be lost at the next reboot. To make them persistent, you
+     need to either:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       <command>chroot</command> to the system root <filename>/</filename> file
+       system from the dracut emergency shell. You may need to mount the
+       <filename>/proc</filename>, <filename>/sys</filename>, and
+       <filename>/dev</filename> partitions as well if dracut needs access to
```

Comment:
it is not dracut that needs access to these filesystems but the commands that you execute in the chroot. dracut already has them mounted in the ramdisk

Reply:
Why don't you just provide a command line for copy and paste? I always use

     for x in proc sys dev run; do mount --bind /$x /mnt/$x; done